src/features/create-drop/contexts/CreateNftDropContext.tsx

```diff
@@ -392,7 +202,7 @@ export const CreateNftDropProvider = ({ children }: PropsWithChildren) => {
     await addToBalance({
       wallet: await window.selector.wallet(),
       amountYocto: totalRequired.toString(),
-      successUrl: window.location.origin + '/drops',
+      successUrl: window.location.origin + '/drop/nft/new',
```

Comment:
Just wanna double check - user will be redirected back to create nft drop page instead of drop manager. Is this intended?